.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,87 @@
+name: "\U0001F41B Bug Report"
+description: Something isn't working as expected
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please answer these questions before submitting your issue. Thanks!
+  - type: textarea
+    id: problem
+    attributes:
+      label: What did you do?
+      description: If possible, provide a recipe for reproducing the error.
+    validations:
+      required: true
+  - type: textarea
+    id: expect
+    attributes:
+      label: What did you expect to see?
+    validations:
+      required: false
+  - type: textarea
+    id: see
+    attributes:
+      label: What did you see instead?
+    validations:
+      required: true
+  - type: textarea
+    id: version
+    attributes:
+      label: Versions of the cluster
+      value: |-
+             DM version (run `dmctl -V` or `dm-worker -V` or `dm-master -V`):
+             ```console
+             (paste DM version here, and you must ensure versions of dmctl, DM-worker and DM-master are same)
+             ```
+             Upstream MySQL/MariaDB server version:
+             ```console
+             (paste upstream MySQL/MariaDB server version here)
+             ```
+             Downstream TiDB cluster version (execute `SELECT tidb_version();` in a MySQL client):
+             ```console
+             (paste TiDB cluster version here)
+             ```
+             How did you deploy DM: DM-Ansible or manually?
```

Comment:
We should taper our support for ansbile
Perhaps it would be better to replace ansbile with `tiup` because there is no ansible-deploy section in official doc

Reply:
I looked up the official doc for deploy DM, and it's recommended to deploy DM with tiup.
But for `manually`, I'm not sure if this needs to be kept.

Reply:
I agree switch to tiup